integration/nwo/standard_networks.go

```diff
@@ -358,3 +333,28 @@ func FullEtcdRaftNoSysChan() *Config {
 
 	return config
 }
+
+func MultiNodeBFTNoSysChan() *Config {
+	config := BasicConfig()
+
+	config.Consensus.Type = "BFT"
+	config.Orderers = []*Orderer{
+		{Name: "orderer1", Organization: "OrdererOrg"},
+		{Name: "orderer2", Organization: "OrdererOrg"},
+		{Name: "orderer3", Organization: "OrdererOrg"},
+	}
+	config.Profiles = []*Profile{
+		{
+			Name:          "TwoOrgsAppChannelBFT",
+			Consortium:    "SampleConsortium",
+			Organizations: []string{"Org1", "Org2"},
+			Orderers:      []string{"orderer1", "orderer2", "orderer3"},
+		},
+	}
+	config.SystemChannel = nil
+	config.Consensus.ChannelParticipationEnabled = true
```

Comment:
we should eventually remove `ChannelParticipationEnabled` option since it's the only way... both here and in the orderer.yaml